evals/elsuite/make_me_say/autoeval.py

```diff
@@ -17,7 +17,7 @@
 )
 
 
-@backoff.on_exception(backoff.constant, InvalidRequestError, max_tries=3)
+@backoff.on_exception(backoff.constant, BadRequestError, max_tries=3)
```

Comment:
this seems like it would replicate the old behavior, but the old behavior doesn't seem correct since it's retrying on a bad request which i'd imagine should give the same result on subsequent tries (?)

Reply:
Yeah, i agree. I was erring on the side of matching previous behavior vs. fixing anything currently broken.

That said, i can update this to `InternalServerError`